### PR TITLE
fix the banner image, meetup listing meetup-15, meetup-16

### PR DESCRIPTION
**Pull request by hardfire**

### CODE REVIEW

**Comment by geshan** [comment]
looks like needs a rebase

**Comment by samundra** [comment]
Review it please and when ready can be merged.

File: _meetup/16-Jan-2016.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: meetup
-title: "Meetup #15 - 16 Jan 2016"
+title: "Meetup #15 - Coming 16 Jan 2016"
```
Comment (geshan) [inline _meetup/16-Jan-2016.md]: I think I messed it pretty bad.

Comment (samundra) [inline _meetup/16-Jan-2016.md]: no it's ok that was me who removed `Coming` and then again added it back :)

Comment (hardfire) [inline _meetup/16-Jan-2016.md]: maybe we should add a logic if the date is in the future, the name attaches UPCOMING automatically. That is much better than this. 
It should be somewhere in this file IMO https://github.com/developers-nepal/php/blob/gh-pages/_layouts/meetup.html

Comment (geshan) [inline _meetup/16-Jan-2016.md]: I thnk it is an overkill for now :) let's do the bare minimum.

Comment (samundra) [inline _meetup/16-Jan-2016.md]: I also think it's overkill for now. It should have worked with the existing code base but looks like we missed something. I will look into this more maybe sometime this weekend. :ok:

**Comment by samundra** [comment]
I have fixed the issue. The page now appears on the list. For now I have removed `date` attribute from `meetup` collection.

File: _includes/footer.html [hunk 2]
```diff
@@ -10,4 +10,7 @@
 <div align="center">
   <img src="{{ site.baseurl }}/public/images/dn-icon.gif"
     alt="Developers Nepal Community">
+<br/>
+<small>Last updated date {{ site.time | date: "%A, %b %d, %Y" }}</small>
```
Comment (samundra) [inline _includes/footer.html:14]: I have added last page update time just above the footer.

File: _layouts/default.html [hunk 3]
```diff
@@ -14,7 +14,6 @@
     </div>
 
     {% include footer.html %}
-
+    <!-- <small>Last generated on : {{ site.time }}</small> -->
```
Comment (hardfire) [inline _layouts/default.html:17]: can we remove this comment ?

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
+## Local Machine Setup for Contribution
+
+- Install ruby
+- Install Jekyll
+- Have some workable knowledge on github pages
+
+Use following Jekyll command to serve locally. Below command was tested on `jekyll 3.1.6`
+
+```bash
+$ jekyll serve watch
```
Comment (geshan) [inline README.md:13]: +1

File: _layouts/default.html (hunk 3 above)
Comment (geshan) [inline _layouts/default.html:17]: +1

**Comment by geshan** [comment]
merging this, then I will fix the commented code issue in `_layouts/default.html`

**Pull request merged**